examples/advanced/create-react-app/src/index.scss

```diff
@@ -0,0 +1,11 @@
+body {
+  margin: 0;
+  padding: 0;
+  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen", "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue", sans-serif;
```

Comment:
Should we have this declaration and the code font-family below use style dictionary values?

Reply:
No, because this is the CRA index file, so I wanted to keep it exactly the same as the original CRA. Also, keep in mind that the app has not been ejected, so we can easily update it in the future.

Reply:
Gotcha, that makes sense